bitnami/postgresql-repmgr/13/debian-11/rootfs/opt/bitnami/scripts/librepmgr.sh

```diff
@@ -560,13 +630,35 @@ EOF
 #########################
 repmgr_wait_primary_node() {
     local return_value=1
-    local -i timeout=60
-    local -i step=10
-    local -i max_tries=$((timeout / step))
+    local -i timeout="${REPMGR_WAIT_PRIMARY_TIME:-60}"
+    local -i randomize=$(( ( RANDOM % 20 )  + 1 ))
```

Comment:
What is the point of randomizing the `max_retries`?

Randomizing the reconnection interval could make sense to avoid collisions, but I don't see the point of setting a random number of retries.

In addition, `max_retries` is not used for anything except for the debug message below, the loop condition will still retry $(timeout / step) times.

Reply:
You are right at this point, I will do a new commit with this corrected.
I will use your native method to randomize the step variable and remove additional default values. 
REPMGR_WAIT_PRIMARY_TIME and REPMGR_WAIT_PRIMARY_STEP are already declared in the postgresql-env.sh file.